### ISSUE TITLE
Could you please check the entry request of SportsMOT in CodaLab?

### ISSUE THREAD

**Issue by bamboopu** [MCG-NJU/SportsMOT#4]
Thanks for your awesome work!
I want to evaluate my MOT algorithm on SportsMOT to further improve it. However, the challenge has already ended. Could you please check the entry request of SportsMOT in CodaLab? I would highly appreciate it. :)

**Comment by ZXYFrank** [comment]
Thanks! 
The competition, i.e. ECCV 2022 DeeperAction workshop, has already ended. We do not plan to release the test set, which may lead to algorithms designed to overfit the dataset.
The concerned properties,  such as split ratio, annotation quality and scene diversity, 
are checked to be the same for SportsMOT's train, validation and test set. Feel free to design algorithms using train and val. If your algorithms has good generalization ability, testing will not be troublesome.
Keep track of our repo. We will update related challenge info here :)

**Comment by bamboopu** [comment]
I get the approach to the train and val dataset! Many thanks for your quick reply. Best wishes.

**Comment by bamboopu** [comment]
I get the approach to the train and val dataset! Many thanks for your quick reply. Best wishes.
